### ISSUE TITLE
[Zealot] Chaotic Agitation

### ISSUE THREAD

**Issue by Laurentz18** [WarEmu/WarBugs#11384]
Expected behavior and actual behavior:
Chaotic Agitation is a channeled Zealot AOE pulsing ability centered on a target.

![laurentz_017](https://user-images.githubusercontent.com/36253097/36654667-d9043d08-1a83-11e8-923a-9bc2f14e8ce6.jpg)

The target becomes the center point of a magical explosion that also simultaneously hits anyone within 30ft of them. At present, if the target of Chaotic Agitation Disrupts or Blocks the hit on them, it also prevents anyone else from being hit. In Live, it did not matter if the central target Disrupted/Blocked the pulse, everyone else within 30ft would be hit and they also could Disrupt/Block the hit on themselves. The spell is not supposed to channel through the target and on to others, but rather is calling the blast down with the target merely acting as the center of the blast radius. 

Steps to reproduce the problem:
This effect can be demonstrated most easily by attempting to use it on a healer or tank with a high tendency to Disrupt or Block the attack.  

I do not have any good screenshots or video at this time, but will try to generate some.

**Comment by sioding** [comment]
Yes please add screens and you said how it was on live - please add a proof for this like a video.

**Comment by Laurentz18** [comment]
Ok I've been parsing through the handful of dps Zealot videos on youtube that exist from after the main Harbinger ability change and have a moment of video that backs this up. You will probably have to rewatch several times while advancing frames one at a time manually since there is an enormous amount of info on the screen to digest. Fortunately, they are using mods that display useful icons here. The Chaotic Agitation icon is light blue with what looks like a black spiderweb on it.

The clearest example is in the video Ode to Wind of Insanity (which may also help me prove how WOI is bugged, but more on that later). At 3:10 the zealot starts channeling Chaotic Agitation on a warrior priest. The channel bar shows the spell and you see a disrupt appear on the warrior priest, but then you see damage/absorb simultaneously going off on the other Order in range. 

https://www.youtube.com/watch?v=CLno108Cjjg

I'm still frame by frame analyzing another vid, will post if it has anything that is clear.

**Comment by RoR-Zomega** [comment]
It's coming up on 2 years for this one. In case any devs still need confirmation I've extracted the 2 frames from the linked video that matter. The first frame is the frame before the ability ticks (you can see the Warrior Priest selected as the target) and the second frame is the frame directly after showing the Warrior Priest disrupting it along with damage/absorb numbers on people within the AoE.

Frame 1:

![CA1](https://user-images.githubusercontent.com/60266177/73776156-2a0af000-477f-11ea-9993-68f34dea573f.PNG)

Frame 2:

![CA2](https://user-images.githubusercontent.com/60266177/73776165-2e370d80-477f-11ea-94d0-22997a3ef829.PNG)
